### PR TITLE
Added documentation for promoting document set version apis

### PR DESCRIPTION
**Pull request by swapnil1993**

### CODE REVIEW

File: changelog/Microsoft.FileServices.json [hunk 1]
```diff
@@ -2648,6 +2648,48 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the [documentSetVersion](https://docs.microsoft.com/en-us/graph/api/resources/documentSetVersion?view=graph-rest-1.0) resource",
+          "Target": "documentSetVersion"
+        },
+        {
+          "Id": "b7269ffc-d6e7-46fb-8c8b-af9931aa869b",
+          "ApiChange": "Resource",
+          "ChangedApiName": "documentSetVersions",
+          "ChangeType": "Addition",
+          "Description": "Added the **documentSetVersions** relationship to the [listItem](https://docs.microsoft.com/en-us/graph/api/resources/listItem?view=graph-rest-1.0) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.FileServices.json:2675]: Your PR doesn't include this change. Please add the documentSetVersions relationship to the listItem resource type.

**Comment by JarbasHorst** [comment]
Hello @swapnil1993, I reviewed your PR and left one comment. Please, take a look when you will have time.

**Also, to avoid merge conflicts, please merge PR [17856](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17856) into your branch to include my fixes and before addressing my comments.**

**Comment by JarbasHorst** [comment]
@Lauragra / @isvargasmsft  this PR is ready for final review. Please, merge PR [17873](https://github.com/microsoftgraph/microsoft-graph-docs/pull/17873) into this branch to include my final edits.

Also, please add the label "content review in progress" to this PR. Thanks!

**Comment by swapnil1993** [comment]
@Lauragra @isvargasmsft The PR is pending review for 2 weeks, please take a look. The schema is already live for this change in prod.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**